prow/manifests/overlays/metal3/config.yaml

```diff
@@ -1532,6 +1539,22 @@ presubmits:
                 value: "TRUE"
             image: docker.io/pipelinecomponents/markdownlint:0.13.0@sha256:9c0cdfb64fd3f1d3bdc5181629b39c2e43b6a52fc9fdc146611e1860845bbae0
             imagePullPolicy: Always
+    - name: markdownlint
```

Comment:
Just add `release-24.0` on L1512, don't copy the config at all.

Reply:
done